src/gatsby/src/templates/park.js

```diff
@@ -621,7 +608,28 @@ export const query = graphql`
           facilityName
           facilityCode
           isActive
-          isCamping
+          icon
+          iconNA
+          rank
+          defaultDescription {
+            data
+          }
+          appendStandardCalloutText {
+            data
+          }
+        }
+      }
+      parkCampingTypes {
+        isActive
+        isCampingOpen
+        hideStandardCallout
+        description {
+          data
+        }
+        campingType {
+          campingTypeName
+          campingTypeCode
+          isActive
           icon
```

Comment:
I was able to see parkCampingTypes.campingType through API but not through GraphQL